sdcm/keystore.py

```diff
@@ -101,6 +101,9 @@ def get_argusdb_credentials(self):
     def get_argus_rest_credentials(self):
         return self.get_json("argus_rest_credentials.json")
 
+    def get_baremetal_credentials(self):
```

Comment:
this name is misleading, since it's more then just credentials.
it's also the information of a specific set of baremetal setup

Reply:
Will change

Reply:
Done